DOCUMENTATION.md

```diff
@@ -338,6 +339,21 @@ content-cli list packages -p <your-chosen-profile>
 content-cli list packages -p <profileName> --json --packageKeys <package1> <package2>
 ```
 
+### List all packages across flavors
+
+With this command you can retrieve a list of all packages across flavors within a team.
+The command takes your permissions into consideration and only lists the
+packages you have access to.
+
+-   It is also possible to download packages in JSON format by adding '--json' option.
+-   When the JSON format option is used, it is also possible to include package dependencies by adding '--withDependencies' flag.
+-   When the JSON format option is used, it is also possible to filter packages by adding '--packageKeys' parameter
+
+```
+content-cli list exportPackages -p <your-chosen-profile>
```

Comment:
We were discussing with @promeris how should we name these new commands and were thinking that maybe we could do something like this:
```
content-cli config list 
content-cli config export
content-cli config import
```
What do you think?